docker/run-test

```diff
@@ -11,16 +11,44 @@ if [[ ${CI_FS_MTR} == 'yes' ]]; then
     CI_FS_DOCKER_FLAG="--mount type=bind,source=/mnt/ci_disk_dir_${CMAKE_BUILD_TYPE},destination=/tmp/ci_disk_dir"
 fi
 
+if [[ "${KEYRING_VAULT_MTR}" == 'yes' ]] && [[ ${DOCKER_OS} != 'centos:6' ]] && [[ ${DOCKER_OS} != 'i386/centos:6' ]]; then
+    if [[ $(docker network list | grep bridge-vault) != *bridge-vault* ]]; then
+        docker network create bridge-vault
+    fi
+
+    # Bootstrap dev mode vault instances
+    BOOTSTRAP_DEV_V1="$(bash $SCRIPTS_DIR/bootstrap-vault --mode=dev --version="${KEYRING_VAULT_V1_VERSION}" --alias=v1 --port="9200" --devtoken=${VAULT_V1_DEV_TOKEN})"
+    export VAULT_V1_DEV_MTR_TOKEN=$(echo "$BOOTSTRAP_DEV_V1" | grep "MTR token" | awk -F ':' '{print $2}' | xargs)
```

Comment:
Any reason why there are no `VAULT_V1_DEV_ROOT_TOKEN`  and `VAULT_V2_DEV_ROOT_TOKEN` exports for the dev mode?

Reply:
Because this token is not generated on the fly and it's exposed as Jenkins Credential, in withCredentals block